src/main/scala/no/ndla/frontpageapi/service/ConverterService.scala

```diff
@@ -66,6 +66,7 @@ object ConverterService {
     val withoutAboutSubject = domain.SubjectFrontPageData(
       None,
       subject.name,
+      subject.filters,
```

Comment:
Hmmm, mangler det ikke å ha med `subject.filters` i api.SubjectPageData` også?

Reply:
Jo. Fikser.

Reply:
Mulig jeg misforstår noe her, men hvis hvis det er på hovedforsiden det skal ligge to lenker til samme subject, men med forskjellige filtre er det vel `FrontPageData` som bør oppdateres (den modellen som returneres når man fetcher frontpage). I dag ser den slik ut: 
```json
{
	"topical": [],
	"categories": [{
			"name": "fellesfag",
			"subjects": ["urn:subject:3"]
		}, {
			"name": "yrkesfag",
			"subjects": []
		}, {
			"name": "studiespesialiserende",
			"subjects": []
		}]
}
```

Men kunne f.eks endret subject til å være en liste av objekter slik:
```json
{
	"topical": [],
	"categories": [{
		"name": "fellesfag",
		"subjects": [
                    {
                       "id": "urn:subject:3",
                       "filters": ["filter1", "filter2"]
                    }
                 ]
		}, {
			"name": "yrkesfag",
			"subjects": []
		}, {
			"name": "studiespesialiserende",
			"subjects": []
		}]
}
```

Reply:
La til databasemigrering for å konvertere fra streng til objekt.